packages/fxa-content-server/server/config/local.json-dist

```diff
@@ -53,7 +53,8 @@
     "simpleRoutes": true,
     "resetPasswordRoutes": true,
     "signUpRoutes": true,
-    "signInRoutes": true
+    "signInRoutes": true,
+    "inlineTotpRoutes":true
```

Comment:
This is a temporary addition to enable this route locally - we are planning to merge these routes into `signInRoutes`.